swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/notifications/NotificationSender.kt

```diff
@@ -220,7 +220,7 @@ class NotificationSender(
       .monitorValue(size())
   }
 
-  private data class NotificationResourceData(
```

Comment:
nit: I'd prefer not leaking this class out. Let's keep it private and build out what the notificationSender requires from the caller code. (Not needed for now but should refactor after testing)